google/cloud/speech/v1/cloud_speech_gapic.yaml

```diff
@@ -0,0 +1,52 @@
+type: com.google.api.codegen.ConfigProto
+generate_samples: true
+language_settings:
+  java:
+    package_name: com.google.cloud.speech.v1
```

Comment:
Java currently puts vkit under spi.{MAJOR_VERSION} (I realized I forgot to mention this in your other PR)